fighters/common/src/general_statuses/catch/catchdash.rs

```diff
@@ -10,20 +10,28 @@ pub fn install() {
 fn nro_hook(info: &skyline::nro::NroInfo) {
     if info.name == "common" {
         skyline::install_hooks!(
-            status_pre_CatchDash,
-            status_end_CatchDash
+            status_pre_CatchDash_common,
+            status_end_CatchDash,
+            bind_address_call_status_end_CatchDash
         );
     }
 }
 
-#[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_status_pre_CatchDash)]
-unsafe fn status_pre_CatchDash(fighter: &mut L2CFighterCommon) -> L2CValue {
+#[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_status_pre_CatchDash_common)]
+unsafe fn status_pre_CatchDash_common(fighter: &mut L2CFighterCommon) -> L2CValue {
     JostleModule::set_overlap_rate_mul(fighter.module_accessor, 6.666);  // 0.3 (base overlap rate) * 6.666 = 2.0 overlap rate
     call_original!(fighter)
 }
 
 #[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_status_end_CatchDash)]
 unsafe fn status_end_CatchDash(fighter: &mut L2CFighterCommon) -> L2CValue {
     JostleModule::set_overlap_rate_mul(fighter.module_accessor, 1.0);  // reset to 0.3 overlap rate
-    call_original!(fighter)
+    0.into()
+}
+
+#[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_bind_address_call_status_end_CatchDash)]
+unsafe fn bind_address_call_status_end_CatchDash(fighter: &mut L2CFighterCommon, _agent: &mut L2CAgent) -> L2CValue {
+    JostleModule::set_overlap_rate_mul(fighter.module_accessor, 1.0);  // reset to 0.3 overlap rate
```

Comment:
I mean if you're calling `fighter.status_end_CatchDash()` you wouldn't need this line here
otherwise it's fine

Reply:
it's a safeguard, Mr. triple sure

Reply:
I mean it in the sense that if the bind_address version calls the normal one, and the normal one is modified, you'd be making sure every instance of it is being changed

Reply:
oy vey